src/pages/fr/core-concepts/project-structure.md

```diff
@@ -0,0 +1,105 @@
+- [Des composants Front-end (React, etc.)](/fr/core-concepts/framework-components/)
+- [Des fichiers de style (CSS, Sass)](/fr/guides/styling/)
+- [Du Markdown](/fr/guides/markdown-content/)
+
+Astro traite, optimise et regroupe les fichiers `src/` pour créer le site web final qui est délivré au navigateur. Contrairement au répertoire statique `public/`, les fichiers `src/` sont traités et assemblés pour vous par Astro.
+
+Quelques fichiers (comme les composants Astro) ne sont pas envoyés au navigateur tel quel, mais plutôt en temps qu'HTML statique. D'autres fichiers (comme les CSS) sont envoyés au navigateur, mais peuvent être optimisés ou regroupés avec d'autres fichiers CSS pour améliorer les performances.
+
+### `src/components`
+
+**Les composants** sont généralement du code réutilisables pour vos pages HTML. Ils peuvent être des [composants Astro](/fr/core-concepts/astro-components/), ou des [composants frontend](/fr/core-concepts/framework-components/) comme React ou Vue. Il est commun de grouper et d'organiser tous les composants de votre projet dans ce dossier.
+
+C'est une convention commune dans les projets Astro, mais c'est pas obligatoire. Organisez vos composants comme vous le voulez, si vous le souhaitez !
+
+### `src/layouts`
+
+[Les composants Layouts](/fr/core-concepts/layouts/) sont des types de composants particuliers qui s'appliquent à des pages. Ils sont généralement utilisés par les [Pages Astro](/fr/core-concepts/astro-pages/) et les [Pages Markdown](/fr/guides/markdown-content/) pour définir leur mise en page.
+
+Comme `src/components`, organisez vos composants Layouts comme vous le souhaitez, ils ne sont pas obligatoires.
+
+### `src/pages`
+
+[Les composants Pages](/fr/core-concepts/astro-pages/) sont des types de composants particuliers utilisés pour créer de nouvelles pages. Une Page peut être un composant Astro (`.astro`) ou un fichier Markdown (`.md`) qui représente une page de contenu pour votre site.
+
+> ⚠️ `src/pages` les un dossier **obligatoire** dans votre projet Astro. Sans ça, votre site n'aura aucune pages ou routes !
+
+### `src/styles`
+
+C'est une convention commune de stocker vos fichiers CSS ou Sass dans le dossier `src/styles`, mais c'est pas obligatoire. Du moment que vos fichiers de style sont quelque pars dans le dossier `src/`, Astro va les gérer et les optimiser.
+
+### `public/`
+
+Le dossier `public/` est là pour les fichiers et les ressources qui ne doivent pas être traités durant le processus de compilation d'Astro. Ces fichiers seront copiés dans le dossier de compilation sans modification.
+
+Cet interaction fait en sorte que `public/` soit idéal pour les ressources communes comme des images, des polices d'écriture, ou des fichiers spéciaux comme `robots.txt` et `manifest.webmanifest`.
+
+Vous pouvez placer des fichiers CSS et JavaScript dans le dossier `public/`, mais gardez à l'esprit que ces fichiers ne seront pas regroupés ou optimisés dans votre build final.
```

Comment:
"...regroupés ni optimisés..."

Reply:
Not sure about this one, it's right lexically but doesn't feel right in this context i would better change this to a "`et/ou`"

Reply:
This is a style point for me. I say "nor" in English much more than the average English speaker. I find it more clear.